compiler/rustc_interface/src/passes.rs

```diff
@@ -104,8 +104,8 @@ mod boxed_resolver {
     // then resolver_arenas and finally session.
     impl Drop for BoxedResolverInner {
         fn drop(&mut self) {
+            // Drop the resolver before we drop the session and arenas.
             self.resolver.take();
-            self.resolver_arenas.take();
```

Comment:
This code resulted in dropping `resolver_arena` before `session`.
Is this order not necessary?